tensorflow_addons/layers/crf.py

```diff
@@ -27,8 +27,37 @@
 class CRF(tf.keras.layers.Layer):
     """Linear chain conditional random field (CRF).
 
+    Inherits from: `tf.keras.layers.Layer`
+
     References:
         - [Conditional Random Field](https://en.wikipedia.org/wiki/Conditional_random_field)
+
+    Example:
+
+    >>> output = CRF(12)
+    >>> output.get_config() == {'name': 'crf', 'trainable': True, 'dtype': 'float32',
+    ... 'units': 12, 'chain_initializer': {'class_name': 'Orthogonal',
+    ... 'config': {'gain': 1.0, 'seed': None}}, 'use_boundary': True,
+    ... 'boundary_initializer': {'class_name': 'Zeros', 'config': {}}, 'use_kernel': True}
+    True
```

Comment:
The example is not so useful. Can we have
```
layer = CRF(4)
inputs = xxx
layer(inputs)
```
or something like this?

Reply:
@Harsh188 Just a reminder: here is also a comment.

Reply:
@WindQAQ Yep! Currently working on this but I've been running into issues where the format of the output is creating doctest issues.

Reply:
@WindQAQ I've had no luck with the doctest error. I posted a [message](https://gitter.im/tensorflow/sig-addons) on Gitter describing my issue can you take a look at it when you have time? Thanks

Reply:
@Harsh188 

How about

```python
"""
    Example:

    >>> layer = tfa.layers.CRF(units=4)
    >>> inputs = np.random.rand(2, 4, 8).astype(np.float32)
    >>> decoded_sequence, potentials, sequence_length, chain_kernel = layer(inputs)
    >>> decoded_sequence.shape
    TensorShape([2, 4])
    >>> potentials.shape
    TensorShape([2, 4, 4])
    >>> sequence_length
    <tf.Tensor: shape=(2,), dtype=int64, numpy=array([4, 4])>
    >>> chain_kernel.shape
    TensorShape([4, 4])
"""
```

Reply:
That's a lot cleaner than what I have currently thanks :)